Documentation/git-rebase.txt

```diff
@@ -609,6 +609,13 @@ provided. Otherwise an explicit `--no-reschedule-failed-exec` at the
 start would be overridden by the presence of
 `rebase.rescheduleFailedExec=true` configuration.
 
+--keep-decorations::
+--no-keep-decorations::
+	Automatically force-update any branches that point to commits that
+	are being rebased. Any branches that are checked out in a worktree
+	or point to a `squash! ...` or `fixup! ...` commit are not updated
+	in this way.
+
```

Comment:
This is a great idea! I'm not sure I would call this `keep-decorations`, though, I'd probably go for `update-corresponding-local-refs` or something.

Reply:
Perhaps `--update-refs` as a shorter option?

Reply:
That sounds good, too!